spec/factories/company_values.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :company_value do
+    title { 'title company' }
```

Comment:
I would change this to something more clear, like helpful

Reply:
I bet on happy :)